### DISCUSSION TITLE
Gopro Panning

### DISCUSSION THREAD

**Discussion by Pax709** [gopro/labs#503] [Q&A]
Hello, I would like to ask for an explanation of this problem, is it really the direction of panning and using linear mode that affects video skipping, or is it due to Hypersmooth?

Link : https://youtu.be/7YbNFRGWszQ?si=S2-gIhXaPKrjQ5ev

**Comment by dnewman-gpsw** [comment]
Hypersmooth has to separate shake and intended movement, no idea how it does it.  Slow pans are hard, as per frame motion is low, like shake.  My guesses anyway.  It gets better with each camera generation. Linear vs Wide is interesting, I always shoot wide and it offers more flexibility in post, including linear.  Also ReelSteady in GoPro Player can smooth out all your pans, as it was designed for cinematic motion, as it can look at the whole file to determine your intent.

**Reply by Pax709** [reply]
Thank you for the answer, that is, using the camera for panning you need to avoid linear mode. When you write about RealSteady, you mean disable HS in the camera and continue stabilizing in Real Steady, but then it is best to have 5.3K 8.7 video footage?

A bit of a shame, as I was just about to use L+HL for the first time on vacation to avoid the extra step in Davinci Resolve.

**Reply by dnewman-gpsw** [reply]
You don't need to disable HS, I never do. While 8:7 adds more options it is not necessary for ReelSteady, RS existed well before 8:7. RS can work on pretty much whatever you shoot, 16:9 or 4:3.  For panning, no extra vertical is even helpful.  I believe there are advantages to combining HS and RS, but they are subtle.  Try it.  You should try on your existing pan test footage.

**Reply by Pax709** [reply]
I've seen how Real Steady works, it can remove shake(in low light conditions), but I guess that's on material without HS on. That is, in vacation mode I will use wide lens + HS On to avoid jerking when panning.

**Reply by dnewman-gpsw** [reply]
RS in GoPro Player works with HS on or off.

**Reply by Pax709** [reply]
The point is that some recommend turning off Hypersmooth altogether to make the final better in RS.

**Reply by dnewman-gpsw** [reply]
Making sure it is clear that is no longer a requirement, as it was initially.  The small advantage to having HS off, is the potential for slightly more FOV, the small advantage to leaving HS on is higher quality images due to more efficient compression.
